topics/proteomics/tutorials/clinical-mp-data-verification/tutorial.md

```diff
@@ -0,0 +1,406 @@
+>        - *"Variable modification(s)"*: `2: Oxidation of M [15.99491461956]`
+>        - *"Use more stringent criterion for unrestricted modification searching"*: `Yes`
+>        - *"Consider amino acid substitution modifications?"*: `Yes`
+>    - In *"Digestion"*:
+>        - *"Enzyme"*: `Trypsin`
+>        - *"Max Missed Cleavages"*: `2`
+>    - In *"Mass spectrometer"*:
+>        - In *"Tolerance"*:
+>            - *"Precursor Tolerance"*: `10`
+>            - *"Precursor Unit"*: `ppm`
+>            - *"Tolerance"*: `0.6`
+>        - In *"PSM"*:
+>            - *"Fragmentation Method"*: `CID/HCD`
+>            - *"Scoring Method"*: `HyperScore`
+>            - *"Minimum Charge"*: `2`
+>            - *"Maximum Charge"*: `6`
+>
+>
+{: .hands_on}
+
+# Collapsing all the data
+
+Remember that PepQuery2 generates a PSM Rank file for each input MGF file, so we will have four PSM Rank files. To make the analysis more efficient, we will collapse these four PSM Rank files into one dataset.
+
+
+> <hands-on-title> Collasping PSM rank files into a singular dataset using Collapse Collection </hands-on-title>
+>
+> 1. {% tool [Collapse Collection](toolshed.g2.bx.psu.edu/repos/nml/collapse_collections/collapse_dataset/5.1.0) %} with the following parameters:
+>    - {% icon param-file %} *"Collection of files to collapse into single dataset"*: `psm_rank_txt` (output of **PepQuery2** {% icon tool %})
+>    - *"Keep one header line"*: `Yes`
+>
+>
+{: .hands_on}
+
+
+
+## Filtering out confident peptides from PepQuery2 with **Filter**
+
+Now, we want to filter for confident peptides from PepQuery2 and prepare them for the Query Tabular tool.
+
+> <hands-on-title> Filter </hands-on-title>
+>
+> 1. {% tool [Filter](Filter1) %} with the following parameters:
+>    - {% icon param-file %} *"Filter"*: `output` (output of **Collapse Collection** {% icon tool %})
+>    - *"With following condition"*: `c20=='Yes'`
+>    - *"Number of header lines to skip"*: `1`
+>
+>
+{: .hands_on}
+
+
+> <hands-on-title> Remove header line from filtered PepQuery peptides with Remove beginning </hands-on-title>
+>
+> 1. {% tool [Remove beginning](Remove beginning1) %} with the following parameters:
+>    - {% icon param-file %} *"from"*: `out_file1` (output of **Filter** {% icon tool %})
+>
+>
+{: .hands_on}
+
+
+> <hands-on-title> Cut (select out) peptide sequences from PepQuery output with Cut </hands-on-title>
+>
+> 1. {% tool [Cut](Cut1) %} with the following parameters:
+>    - *"Cut columns"*: `c1`
+>    - {% icon param-file %} *"From"*: `out_file1` (output of **Remove beginning** {% icon tool %})
+>
+>
+{: .hands_on}
+
+
+##  **Query Tabular**
+
+We will use the Query Tabular tool { %cite Johnson2019 % } to search the PepQuery-verified peptides against the concatenated dataset that contains peptides and proteins from SearchGUI/Peptide and MaxQuant.
```

Comment:
this explanation is very technical ..we are searching our peptides against output from these tools, but *why* are we doing that, what characteristic of the peptides are we looking for here? (I am not a proteomics person, but try to write the tutorial in such a way that people who are less familiar with the tools and topic can also understand it).